tests/pytest.ini

```diff
@@ -10,5 +10,7 @@ markers:
     sanity_check: override the default sanity check settings
     topology: specify which topology testcase can be executed on: (t0, t1, ptf, etc)
     platform: specify which platform testcase can be executed on: (physical, virtual, broadcom, mellanox, etc)
+    completeness_level: specify test completeness (coverage) level (Debug, Basic, Confident, Thorough)
+    supported_completeness_level: test supported levels of completeness (coverage) level (Debug, Basic, Confident, Thorough)
```

Comment:
I don't feel necessary for test to declare supported levels.

Reply:
Declared supported levels can be used for - 1) Global normalization. 2) If the test needs to supersede the module level of completeness.

Reply:
For example, if a testcase has following supported markers:
```
pytestmark = [
    pytest.mark.supported_completeness_level(CompletenessLevel.basic, CompletenessLevel.thorough)
]
```
If this testcase is executed with CLI argument `--completeness_level=confident`, normalization is needed to decide what level test should be run at.
In this case, the normalization logic pulls all the supported levels from the testcase during ` pytest_runtest_setup`, and decides the normalized value should be `basic`.

The testcase will finally be run in `basic` completeness level.